etc/config/c++.amazon.properties

```diff
@@ -1899,6 +1899,8 @@ compiler.esp32g2021r2.exe=/opt/compiler-explorer/xtensa/xtensa-esp32-elf-gcc8_4_
 compiler.esp32g2021r2.name=Xtensa ESP32 gcc 8.4.0 (2021r2)
 compiler.esp32g2022r1.exe=/opt/compiler-explorer/xtensa/xtensa-esp32-elf-gcc11_2_0-esp-2022r1/bin/xtensa-esp32-elf-g++
 compiler.esp32g2022r1.name=Xtensa ESP32 gcc 11.2.0 (2022r1)
+compiler.esp32g20230208.exe=/opt/compiler-explorer/xtensa/xtensa-esp32-elf-12.2.0_20230208/bin/xtensa-esp32-elf-g++
```

Comment:
You'll need to add the new ids to their corresponding group's `compiler` property for these to have any effect.

There's supposed to be a warning against this iirc, wonder why it's not triggering here 🤔

Reply:
This could also be triggered in tests? I ran them as recommended prior to open the MR. I'll update the MR later today.

Reply:
Update on diagnostics:  Once I added the missing ids the pre-commit hooks were able to find some misconfigured ids in the C file. No error reported on C++(and I couldn't catch any by inspection)